document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/mapping/smallfull/model/ixbrl/balancesheet/otherliabilitiesandassets/OtherLiabilitiesAndAssets.java

```diff
@@ -0,0 +1,159 @@
+package uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.gson.Gson;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.AccrualsAndDeferredIncome;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.CreditorsAmountsFallingDueAfterMoreThanOneYear;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.CreditorsAmountsFallingDueWithinOneYear;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.CurrentNetCurrentAssets;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.CurrentTotalAssetsLessCurrentLiabilities;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.PrepaymentsAndAccruedIncome;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.PreviousTotalAssetsLessCurrentLiabilities;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.balancesheet.otherliabilitiesandassets.items.ProvisionForLiabilities;
+
+import java.util.Objects;
+
+public class OtherLiabilitiesAndAssets {
+
+    @JsonProperty("prepayments_and_accrued_income")
+    private PrepaymentsAndAccruedIncome prepaymentsAndAccruedIncome;
+
+    @JsonProperty("creditors_amounts_falling_due_within_one_year")
+    private CreditorsAmountsFallingDueWithinOneYear creditorsAmountsFallingDueWithinOneYear;
+
+    @JsonProperty("current_net_current_assets")
+    private CurrentNetCurrentAssets currentNetCurrentAssets;
+
```

Comment:
could we add `previous_net_current_assets`?
```
    @JsonProperty("previous_net_current_assets")
    private PreviousNetCurrentAssets previousNetCurrentAssets;
```

Reply:
added